acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/MetricsAcceptanceTest.java

```diff
@@ -51,13 +51,11 @@
   @Test
   void filecoinApisAreCounted() {
     final SignerConfiguration signerConfiguration =
-        new SignerConfigurationBuilder().withMetricsEnabled(true).build();
+        new SignerConfigurationBuilder().withMetricsEnabled(true).withMode("filecoin").build();
```

Comment:
does it make sense to create constants (or enum) for Modes in DSL? its a nit-pick, so feel free to keep it as it is.

Reply:
going to keep it as is ... an enum would simplify finding bugs at compile time rather than test-time ... but not sure it offers a huge speed up over the course of a dev cycle.